### ISSUE TITLE
Are labeled statements supported?

### ISSUE THREAD

**Issue by niwinz** [cesanta/v7#562]
Labeled statements are part of ecmascript3 as mentioned here:
https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Statements/label but it seems that v7 does not supports it.

PD: I'm trying to run clojurescript compiled code on v7 and the generated code uses labeled statements in some cases...

**Comment by dimonomid** [comment]
No, sorry, not yet; we keep postponing implementation of labelled statements.
